<?xml version="1.0"?>

<launch>
    <include file="$(find vrpn_client_ros)/launch/sample.launch">
        <arg name="server" value="192.168.1.200" />
    </include>
    <include file="$(find mavros)/launch/px4.launch">
       <arg name="fcu_url" value="/dev/ttyTHS0:921600" />
    </include>
    <node pkg="px4_zxz" type="single_offboard" name="single_offboard" output="screen">
       <param name="vrpn_topic" value="vrpn_client_node/UAV3/pose" />
    </node>

</launch>
